### PR TITLE
Localizely: Translations update

### PR DESCRIPTION
**Pull request by matibat**
Exported the most recent translations from Localizely

### CODE REVIEW

**Comment by karlprieb** [comment]
With the new Flutter ARB syntax, we lost some plurals for Mandarin and Cantonese. Knowing that the follow six keys had their translations simplified and more generic for Mandarin and Cantonese:
- filesWillBeUploadedPublicly
- filesTooLarge
- foldersWithTheSameNameAlreadyExists
- duplicateFolders
- duplicateFiles
- filesWithTheSameNameAlreadyExists

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -90,6 +94,10 @@
   "@back": {
     "description": "Go back"
   },
+  "backButtonOnboarding": "Back",
+  "@backButtonOnboarding": {
+    "description": "Text of the back button onboarding"
+  },
```
Comment (matibat) [inline lib/l10n/app_en.arb:100]: I would make `back` to be `goBack`, and `backButtonOnBoarding` to be `back`. So we would have `goBack`, `back`, and `backEmphasized`. Although it's not so easy to apply: we would have to change every ARB file and update the Localizely Dashboard.

Why: we don't care about WHERE in the app the text is being used, but WHAT is it for.

File: lib/l10n/app_en.arb [hunk 2]
```diff
@@ -240,6 +258,10 @@
   "@copyRootFolderTxID": {
     "description": "Copy transaction ID of Root Folder tooltip"
   },
+  "copyTooltip": "Copy",
+  "@copyTooltip": {
+    "description": "Tooltip for copy button"
+  },
```
Comment (matibat) [inline lib/l10n/app_en.arb:264]: Same comment: we don't have to mention that it is gonna be used in a tooltip. I would call it `copy` alone and the description would explain what "copy" means: "To copy a text"

File: lib/l10n/app_en.arb [hunk 3]
```diff
@@ -258,6 +280,10 @@
   "@create": {
     "description": "Create an drive"
   },
+  "createAndConfirmPassword": "Please create and confirm your password. You will use this password to log in.",
+  "@createAndConfirmPassword": {
+    "description": "Form title to create a password"
```
Comment (matibat) [inline lib/l10n/app_en.arb:285]: Same: it's talking about WHERE and not WHAT
```suggestion
    "description": "Asks the user to create a password"
```

File: lib/l10n/app_en.arb [hunk 4]
```diff
@@ -308,6 +334,10 @@
   "@creatingFolderEmphasized": {
     "description": "Folder is being created. Emphasized with upper case"
   },
+  "darkMode": "DARK MODE",
+  "@darkMode": {
+    "description": "DARK MODE"
+  },
```
Comment (matibat) [inline lib/l10n/app_en.arb:340]: The standard we chose for any text that's CAPITALIZED is to call it "emphasized" and to explain in the description: "Emphasized with upper case".
```suggestion
  "darkModeEmphasized": "DARK MODE",
  "@darkModeEmphasized": {
    "description": "Refers to the dark theme of the app. Emphasized with upper case"
  },
```

File: lib/l10n/app_en.arb [hunk 5]
```diff
@@ -592,6 +642,10 @@
   "@fileFailedToDownloadFileAboveLimit": {
     "description": "Download failure message when a file is too big"
   },
+  "fileFailedToDownloadFileAbovePublicLimit": "The file(s) you have selected is too large to download",
+  "@fileFailedToDownloadFileAbovePublicLimit": {
+    "description": "Download failure message when a file is too big"
+  },
```
Comment (matibat) [inline lib/l10n/app_en.arb:648]: Can't we use plurals here?

File: lib/l10n/app_en.arb [hunk 6]
```diff
@@ -956,6 +1014,10 @@
   "@letsGetStartedEmphasized": {
     "description": "Let's start setting up the account. Emphasized with upper case"
   },
+  "lightMode": "LIGHT MODE",
+  "@lightMode": {
+    "description": "LIGHT MODE"
+  },
```
Comment (matibat) [inline lib/l10n/app_en.arb:1020]: Same comment than `darkMode`.

**Review by matibat — COMMENTED** [review]
Left some comments, none of them are blockers but maybe tech debts

**Pull request closed without merging**